Src/BootCamp.Chapter/CleanTheFile.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    class CleanTheFile
+    {
+        public void Clean(string file, string outputFile)
+        {
+            if (Testers.IsThisStringValid(file) || Testers.IsThisStringValid(outputFile))
+            {
+                throw new ArgumentException();
+            }
+
+            string cleanFile = "";
+
+            using (var fileS = new FileStream(file, FileMode.Open))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this code to make sure 'fileS' is disposed only once.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)